diff --git a/platformplugin/linux.pri b/platformplugin/linux.pri
index a6899a7..f02c616 100644
--- a/platformplugin/linux.pri
+++ b/platformplugin/linux.pri
@@ -66,5 +66,5 @@ contains(QT_CONFIG, xcb-qt) {
 exists($$PWD/libqt5xcbqpa-dev/$$VERSION) {
     INCLUDEPATH += $$PWD/libqt5xcbqpa-dev/$$VERSION
 } else {
-    error(Not support Qt Version: $$VERSION)
+    INCLUDEPATH += /usr/include/qtxcb-private
 }
