commit 06d9241c88cbdd20aa774b00f0c507a653b667b4
Author: William Floyd <william.png2000@gmail.com>
Date:   Tue Mar 20 18:27:57 2018 -0400

    dde-file-manager: Remove autostart install.

diff --git a/dde-file-manager/dde-file-manager.pro b/dde-file-manager/dde-file-manager.pro
index 1d941346..ab411a11 100644
--- a/dde-file-manager/dde-file-manager.pro
+++ b/dde-file-manager/dde-file-manager.pro
@@ -97,7 +97,6 @@ isEqual(ARCH, sw_64) | isEqual(ARCH, mips64) | isEqual(ARCH, mips32) {
 }else{
     autostart.path = /etc/xdg/autostart
     autostart.files = dde-file-manager-xdg-autostart.desktop dde-file-manager-dialog-autostart.desktop
-    INSTALLS += autostart
 }
 
 win32:CONFIG(release, debug|release): LIBS += -L$$OUT_PWD/../dde-file-manager-lib/release -ldde-file-manager
