diff --git a/configure b/configure
index c0d7609..4119a91 100755
--- a/configure
+++ b/configure
@@ -135,8 +135,10 @@ while [ -n "`echo $1 | grep '-'`" ]; do
 done
 
 cd "$BASEDIR"
-replace "s|unix:LIBS +=.*$|unix:LIBS += -Wl,-rpath,. -Wl,-rpath,$LIBDIR -L$LIBDIR -ltreefrog|" defaults/appbase.pri
-replace "s|unix:INCLUDEPATH +=.*$|unix:INCLUDEPATH += $INCLUDEDIR|" defaults/appbase.pri
+replace "s|unix:LIBS +=.*$|unix:LIBS += -Wl,-rpath,. -Wl,-rpath,$LIBDIR -L$LIBDIR -ltreefrog
+|" defaults/appbase.pri
+replace "s|unix:INCLUDEPATH +=.*$|unix:INCLUDEPATH += $INCLUDEDIR
+|" defaults/appbase.pri
 
 if [ -n "$ENABLE_DEBUG" ]; then
   OPT="$OPT CONFIG+=debug"
@@ -165,21 +167,6 @@ cd "$BASEDIR/tools"
 $QMAKE -recursive $OPT target.path=\"$BINDIR\" header.path=\"$INCLUDEDIR\" datadir=\"$DATADIR\" lib.path=\"$LIBDIR\"
 make qmake
 
-# compile MongoDB files
-cd "$BASEDIR/3rdparty/mongo-c-driver"
-echo
-echo -n "Compiling MongoDB driver library ... "
-$QMAKE $OPT mongo-c-driver.pro
-make clean >/dev/null 2>&1
-make >/dev/null 2>&1
-RET=$?
-if [ $RET != 0 ]; then
-  echo "failed"
-  echo "MongoDB driver not available."
-  exit $RET
-fi
-echo "OK"
-
 echo
 echo "First, run \"make\" and \"sudo make install\" in src directory."
 echo "Next, run \"make\" and \"sudo make install\" in tools directory."
