diff --git a/src/corelib.pro b/src/corelib.pro
index d111220..137e78a 100644
--- a/src/corelib.pro
+++ b/src/corelib.pro
@@ -366,7 +366,7 @@ greaterThan(QT_MAJOR_VERSION, 4) {
 
 
 # Files for MongoDB
-INCLUDEPATH += ../3rdparty/mongo-c-driver/src/mongoc ../3rdparty/mongo-c-driver/src/libbson/src/bson
+INCLUDEPATH += /usr/include/libbson-1.0 /usr/include/libmongoc-1.0
 windows {
 #  DEFINES += MONGO_STATIC_BUILD
 
@@ -384,7 +384,7 @@ windows {
     }
   }
 } else {
-  LIBS += ../3rdparty/mongo-c-driver/libmongoc.a
+  LIBS += -lmongoc-1.0
 }
 #DEFINES += MONGO_HAVE_STDINT
 

