diff --git a/Makefile b/Makefile
index 19ac075..9c16327 100644
--- a/Makefile
+++ b/Makefile
@@ -60,10 +60,10 @@ install:
 
 sass:
 	mkdir -p PageRoot/www/style
-	sass --style compressed --unix-newlines PageRoot/www/scss/reader.scss PageRoot/www/style/reader.css
-	sass --style compressed --unix-newlines PageRoot/www/scss/index.scss PageRoot/www/style/index.css
-	sass --style compressed --unix-newlines PageRoot/www/scss/content.scss PageRoot/www/style/content.css
-	sass --style compressed --unix-newlines PageRoot/www/scss/reset.scss PageRoot/www/style/reset.css
+	sassc --style compressed PageRoot/www/scss/reader.scss PageRoot/www/style/reader.css
+	sassc --style compressed PageRoot/www/scss/index.scss PageRoot/www/style/index.css
+	sassc --style compressed PageRoot/www/scss/content.scss PageRoot/www/style/content.css
+	sassc --style compressed PageRoot/www/scss/reset.scss PageRoot/www/style/reset.css
 
 clean:
 	rm -rf symdir/
