diff --git a/platformthemeplugin/main.cpp b/platformthemeplugin/main.cpp
index cad7998..15184db 100644
--- a/platformthemeplugin/main.cpp
+++ b/platformthemeplugin/main.cpp
@@ -18,7 +18,7 @@
 #include <qpa/qplatformthemeplugin.h>
 #include "qdeepintheme.h"
 
-#include <qt5xdgiconloader/2.0.0/private/xdgiconloader/xdgiconloader_p.h>
+#include <qt5xdgiconloader/3.1.0/private/xdgiconloader/xdgiconloader_p.h>
 
 QT_BEGIN_NAMESPACE
 
