diff --git a/Makefile b/Makefile
index e80d612..1a183aa 100644
--- a/Makefile
+++ b/Makefile
@@ -39,10 +39,10 @@ install:
 	install -Dm755 startdde ${DESTDIR}${PREFIX}/bin/startdde
 	mkdir -p ${DESTDIR}${PREFIX}/share/xsessions
 	@for i in $(shell ls misc/xsessions/ | grep -E '*.in$$' );do sed 's|@PREFIX@|$(PREFIX)|g' misc/xsessions/$$i > ${DESTDIR}${PREFIX}/share/xsessions/$${i%.in}; done
-	install -Dm755 dde-readahead/dde-readahead ${DESTDIR}/${PREFIX}/lib/deepin-daemon/dde-readahead
-	install -Dm644 dde-readahead/dde-readahead.service ${DESTDIR}/lib/systemd/system/dde-readahead.service
-	mkdir -p ${DESTDIR}/lib/systemd/system/multi-user.target.wants/
-	ln -s /lib/systemd/system/dde-readahead.service ${DESTDIR}/lib/systemd/system/multi-user.target.wants/dde-readahead.service
+	install -Dm755 dde-readahead/dde-readahead ${DESTDIR}/${PREFIX}/${LIBDIR}/deepin-daemon/dde-readahead
+	install -Dm644 dde-readahead/dde-readahead.service ${DESTDIR}/${LIBDIR}/systemd/system/dde-readahead.service
+	mkdir -p ${DESTDIR}/${LIBDIR}/systemd/system/multi-user.target.wants/
+	ln -s /${LIBDIR}/systemd/system/dde-readahead.service ${DESTDIR}/${LIBDIR}/systemd/system/multi-user.target.wants/dde-readahead.service
 
 clean:
 	-rm -rf ${GOPATH_DIR}
