diff --git a/accounts/user.go b/accounts/user.go
index 5e1bf7fe..358380e7 100644
--- a/accounts/user.go
+++ b/accounts/user.go
@@ -46,7 +46,7 @@ const (
 	defaultLayout         = "us" + layoutDelim
 	defaultLayoutFile     = "/etc/default/keyboard"
 	defaultUserIcon       = "file:///var/lib/AccountsService/icons/default.png"
-	defaultUserBackground = "file:///usr/share/backgrounds/default_background.jpg"
+	defaultUserBackground = "file:///usr/share/backgrounds/deepin/desktop.jpg"
 
 	maxWidth  = 200
 	maxHeight = 200
