diff --git a/Makefile b/Makefile
index 1d295541..459a3fc5 100644
--- a/Makefile
+++ b/Makefile
@@ -77,8 +77,8 @@ test: prepare
 	env GOPATH="${GOPATH}:${CURDIR}/${GOPATH_DIR}" go test -v ./...
 
 install: build translate install-dde-data install-icons
-	mkdir -pv ${DESTDIR}${PREFIX}/lib/deepin-daemon
-	cp out/bin/* ${DESTDIR}${PREFIX}/lib/deepin-daemon/
+	mkdir -pv ${DESTDIR}${LIBDIR}/deepin-daemon
+	cp out/bin/* ${DESTDIR}${LIBDIR}/deepin-daemon/
 
 	mkdir -pv ${DESTDIR}${PREFIX}/share/locale
 	cp -r out/locale/* ${DESTDIR}${PREFIX}/share/locale
